easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.4-foss-2020a.eb

```
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'MakeCp'
+
+name = 'GEMMA'
+version = '0.98.4'
+
+homepage = 'https://github.com/genetics-statistics/GEMMA'
+description = "Genome-wide Efficient Mixed Model Association"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://github.com/genetics-statistics/GEMMA/archive/refs/tags']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4f57a045d3289afaf31f818bf411ac46c5ee6f78ff8c9c4117963ca54e0bb9f0']
+
+builddependencies = [
+    ('Eigen', '3.3.7', '', True),
```

I think this would work? Using the Eigen-3-3-7-GCCcore-9.3.0.eb
```suggestion
    ('Eigen', '3.3.7'),
```

---

Yes, that's just what I inherited.